src/main/java/gov/nasa/cumulus/metadata/aggregator/MetadataAggregatorLambda.java

```diff
@@ -234,6 +239,33 @@ public String PerformFunction(String input, Context context) throws Exception {
 		return returnable.toJSONString();
 	}
 
+	public HashSet<MENDsIsoXMLSpatialTypeEnum> createIsoXMLSpatialTypeSet(JSONArray isoXMLSpatialTypeConfigJSONArray) throws IllegalArgumentException{
+		HashSet<MENDsIsoXMLSpatialTypeEnum> isoSpatialTypes = new HashSet<>();
+		// if not containing isoXMLTypes, then return an empty HashSet
+		if(isoXMLSpatialTypeConfigJSONArray == null || isoXMLSpatialTypeConfigJSONArray.size()==0) {
+			return isoSpatialTypes;
+		}
```

Comment:
default to bbox seems to be very reasonable.  However, it could fail if the collection is Orbit but not GEODETIC or CARTICIAN.   This seems to be a hard thing to decide?   

Maybe a logic of "if it is nc.iso.xml aka MENDs style xml then isoXMLSpatialType must not be null, otherwise throws exception"?   Let's discuss this on Monday